exchange/exchangetest/request-multi-bidders-one-no-resp.json

```diff
@@ -0,0 +1,169 @@
+{
+  "incomingRequest": {
+    "ortbRequest": {
+      "id": "some-request-id",
+      "site": {
+        "page": "test.somepage.com"
+      },
+      "imp": [
+        {
+          "id": "my-imp-id",
+          "video": {
+            "mimes": ["video/mp4"]
+          },
+          "ext": {
+            "appnexus": {
+              "placementId": 1
+            },
+            "audienceNetwork": {
+              "placementId": "some-placement"
+            }
+          }
+        },
+        {
+          "id": "imp-id-2",
+          "video": {
+            "mimes": ["video/mp4"]
+          },
+          "ext": {
+            "appnexus": {
+              "placementId": 2
+            },
+            "audienceNetwork": {
+              "placementId": "some-other-placement"
+            }
+          }
+        }
+      ],
+      "ext": {
+        "prebid": {
+          "targeting": {}
```

Comment:
Just realized, this requests needs the `includebrandcategory` in the `targeting` in order to represent the fix, correct? That way, the processing will trigger `applyCategoryMapping` where you pointed out the crash initially.

Reply:
You are right, fixed!